### ISSUE TITLE
The extension adds a load of stuff to the DOM, making it unreadable

### ISSUE THREAD

**Issue by TestPartners** [qualweb/qualweb-extension#19]
When the extension is enabled, it adds loads of stuff to every element in the DOM. For instance, what used to be a <p> tag is now <p _selector="html > body:nth-child(2) > div:nth-child(4) > div:nth-child(1) > div:nth-child(1) > div:nth-child(1) > div:nth-child(2) > p:nth-child(1)" _cssrules="true">

Is this intentional? If so, is there a way to turn it off? It makes the DOM impossible to read.

**Comment by carlosapaduarte** [comment]
Yes, that is intentional. The extension adds that information to use it during the evaluation. 
The only way to turn it off is to disable the extension.
